org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/OurConsole.java

```diff
@@ -64,21 +68,26 @@
  */
 
 public final class OurConsole extends JScrollPane {
+	final static int menuShortcutKeyMask = Toolkit.getDefaultToolkit().getMenuShortcutKeyMask();
 
 	/** This ensures the class can be serialized reliably. */
-	private static final long	serialVersionUID	= 0;
+	private static final long			serialVersionUID	= 0;
 
 	/** The style for default text. */
-	private final AttributeSet	plain				= style("Verdana", 14, false, Color.BLACK, 0);
+	private final static AttributeSet	plain				= style("Verdana", 14, false, false, false, Color.BLACK, 0);
+
+	/** The style for table text. */
+	private final static AttributeSet	mono				= style("Courier", 14, false, false, false, Color.BLACK,
+			10);
```

Comment:
Automatic code formatters are also evil: this line break (with only '`10);`' on its own line) seems hard to justify.

Reply:
I think code formatters are the only solution to get a consistent style ... :-)